toolkit/components/filewatcher/tests/xpcshell/test_watch_file_creation_single.js

```diff
@@ -27,7 +27,8 @@ add_task(async function test_watch_single_path_file_creation() {
 
   // Instantiate and initialize the native watcher.
   let watcher = makeWatcher();
-  let deferred = Promise.defer();
+/*
```

Comment:
Ah, this was a debug mistake that shouldn't have been committed. I'll fix this shortly.